java/src/main/java/com/bc/ur/URDecoder.java

```diff
@@ -57,4 +57,14 @@ public IllegalStateException resultError() {
     public boolean receivePart(String s) {
         return BCUR.URDecoder_receive_part(ptrObj, s);
     }
+
+    @Override
+    public boolean dispose() {
```

Comment:
consider implementing `AutoCloseable` so this can be called automatically when used with a try-with-resources block

Reply:
also, would be good to have a class level documentation about leaking memory if this is not called

Reply:
@devrandom Sound great. Let me try it.